doc/api.tex

```diff
@@ -5,8 +5,20 @@
 \begin{enumerate} 
   \item Raw XML: The client resolves the reference and delivers an XML block corresponding to the searched models or model components. 
         The client must then extract the search elements by using XPath queries applied to the the MIVOT namespace.
-  \item XML Wrapper class: The XPath strings mentioned above can be wrapped in generic objects providing accessors retrieving model nodes by selecting types or roles. 
-  \item Model classes: For the most popular models and especially for the models that provide reusable components (Meas/Coord, PhotDM), the API can include objects instantiating model classes (e.g. Photometric filter).
-  \item Automatic extraction of class instances of common frameworks: One of the goal of the data annotation is to facilitate the connection between data and API code. This could be achieved with an API able to automatically build objects from the annotated data without asking the user to infer on meta data. Taking the Astropy example, a model-enable Python API should be able to build e.g. \texttt{Skycoord} instances in a transparent way and feed them with \texttt{FIELD} values from \texttt{TABLE} rows or from \texttt{PARAM}.
+  \item XML Wrapper class: The XPath strings mentioned above can be wrapped in generic objects providing 
+        accessors retrieving model nodes by selecting types or roles. 
+  \item Model classes: For the most popular models and especially for the models that provide reusable 
+        components (Meas/Coord, PhotDM), the API can include objects instantiating model classes (e.g. Photometric filter).
+  \item Automatic extraction of class instances of common frameworks: One of the goal of the data annotation is to 
+        facilitate the connection between data and API code. This could be achieved with an API able to automatically 
+        build objects from the annotated data without asking the user to infer on meta data. Taking the Astropy example, 
+        a model-enable Python API should be able to build e.g. \texttt{Skycoord} instances in a transparent way and 
+        feed them with \texttt{FIELD} values from \texttt{TABLE} rows or from \texttt{PARAM}.
  \end{enumerate}
 
+These different levels should be hidden to the end user. Using MIVOT annotations makes sense only if the client code provides 
```

Comment:
I would say that for practical reasons level 1 and 2 are not very useful and client software should provide level 3 and 4 APIs to end users.

Reply:
I reworded the paragraph,